hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/AbstractParentQueue.java

```diff
@@ -1136,11 +1137,43 @@ public void refreshAfterResourceCalculation(Resource clusterResource,
     CSQueueUtils.updateConfiguredCapacityMetrics(resourceCalculator,
         labelManager.getResourceByLabel(null, clusterResource),
         RMNodeLabelsManager.NO_LABEL, this);
+
+    LOG.info("Refresh after resource calculation (PARENT) {}\n"
```

Comment:
Since we log in info level from the `AbstractLeafQueue.setupQueueConfigs`:

```
      LOG.info(
          "Initializing " + getQueuePath() + "\n" +
              getExtendedCapacityOrWeightString() + "\n"
              + "absoluteCapacity = " + queueCapacities.getAbsoluteCapacity()
              + " [= parentAbsoluteCapacity * capacity ]" + "\n"
```

which contains invalid information in the new queue calculation mode (because the calculation was not done yet), I think it's better to log the updated values on info level too.

Reply:
Ah okay there's another one on INFO. Okay, let's keep this as it is.